src/ViewModel/Converter/ProfileContentHeaderProfileConverter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace eLife\Journal\ViewModel\Converter;
+
+use eLife\ApiSdk\Model\Profile;
+use eLife\Journal\Helper\Callback;
+use eLife\Patterns\ViewModel;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+final class ProfileContentHeaderProfileConverter implements ViewModelConverter
+{
+    private $urlGenerator;
+
+    public function __construct(UrlGeneratorInterface $urlGenerator)
+    {
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    /**
+     * @param Profile $object
+     */
+    public function convert($object, string $viewModel = null, array $context = []) : ViewModel
+    {
+        $affiliations = array_unique($object->getAffiliations()
+            ->map(Callback::method('getName'))
+            ->map(Callback::apply('end'))
+            ->toArray());
+
+        $emailAddress = $object->getEmailAddresses()[0];
```

Comment:
and if there aren't visible email addresses? https://github.com/elifesciences/api-raml/blob/develop/src/model/profile.v1.yaml#L11 doesn't specify a minimum length for the array

Reply:
It's just be `null`.